Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM registry.redhat.io/rhscl/python-38-rhel7
+
 COPY Pipfile Pipfile.lock manage.py run.py ${APP_ROOT}/src/
+
 COPY ros ${APP_ROOT}/src/ros/
 COPY migrations ${APP_ROOT}/src/migrations/
 RUN pip install --upgrade pip && \
     pip install pipenv && \
     pipenv install --system --deploy --ignore-pipfile
-CMD bash -c 'python manage.py db upgrade && python run.py'
```

Comment:
If we are planning to remove `CMD`. Then there should be a readme which tells how to run the container using docker. I mean we will need to pass `CMD` in `docker run` command explicitly. Also, we will need to add [command and args ](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/)in e2edeploy so that when Openshift spins up the container there should be an initial process to run.